prisma/migrations/20231205102037_rename_content_to_text/migration.sql

```diff
@@ -0,0 +1,9 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `content` on the `Article` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "Article" DROP COLUMN "content",
```

Comment:
This will cause us to loose all information previously stored in the `content` column. Consider renaming this column instead of dropping it.